plugins/slack_app/app/helpers/slack_app_helper.rb

```diff
@@ -0,0 +1,12 @@
+require 'uri'
+
+module SlackAppHelper
+  def oauth_url(scopes)
+    redir_url = URI.encode url_for(controller: :slack_app, action: :oauth, only_path: false)
```

Comment:
Nice tip on `to_query`. Can you help me out with the routing stuff? These aren't mounted as resources, so I'm not sure how to get to `/slack_app/oauth` in a more succinct way (fairly new to Rails here).